src/CommonITILTask.php

```diff
@@ -309,7 +309,7 @@ public function assignTechFromtask($input)
             ],
         ];
         foreach ($entities as $entityType => $entityData) {
-            if ($entityType == $input['itemtype']) {
+            if ($entityType == isset($input['itemtype'])) {
```

Comment:
Seems wrong here

You affected the result of ```isset``` function instead of affected ```$input['itemtype']``` value

Reply:
TO be 100% sure, add parenthesis:
```suggestion
            if ($entityType == ($input['itemtype'] ?? '')) {
```

Reply:
As the item type is not given every time, I removed this check and replaced it with if (isset($input[$entityType . 's_id'])) which matches problems_id, changes_id and tickets_id.